bundles/core/src/test/resources/experiencefragment/exporter-experiencefragment3.json

```diff
@@ -0,0 +1,4 @@
+{
+   "experienceFragmentVariationPath":"/content/experience-fragments/language-masters/es/footer/master/jcr:content",

```

Comment:
@vladbailescu 
Should we also add the original configured experienceFragment path apart from the evaluated path of the language specific experience fragment?

Reply:
Yes, I think that might be valuable.